site/docs/v1/tech/apis/reactor.adoc

```diff
@@ -0,0 +1,124 @@
+
+* <<Activate Reactor license>>
+* <<Retrieve Reactor status>>
+* <<Regenerate Reactor license>>
+* <<Deactivate Reactor license>>
+
+
+== Activate Reactor license
+
+This API is used to activate a Reactor license.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Activate a Reactor license
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/reactor#
+--
+
+==== Request Body
+
+[.api]
+[field]#licenseId# [type]#[String]# [required]#Required#::
+The license Id to to activate.
+
+[field]#license# [type]#[String]# [optional]#Optional#::
+The Base64 encoded license value. This value is necessary in an link:https://en.wikipedia.org/wiki/Air_gap_(networking)[air gapped] configuration where outbound network access it not available.
+
+=== Response
+
+This API does not return a JSON response body.
+
+:never_search_error:
+include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:never_search_error!:
+
+== Retrieve Reactor status
+
+This API is used to retrieve the status of Reactor features. This also includes breached password metrics.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve Reactor status
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/reactor#
+--
+
+=== Response
+
+:no_errors:
+:never_search_error:
+include::docs/v1/tech/apis/_standard-get-response-codes.adoc[]
+:no_errors!:
+:never_search_error!:
+
+include::docs/v1/tech/apis/_reactor-response-body.adoc[]
+
+
+== Regenerate Reactor license
+
+This API is used to make requests from FusionAuth to the License server to regenerate a license. This is particularly useful if it suspected that the license key has been compromised and a new one should be regenerated.
+
+Regenerating the license will cause any other instance using the license to fall back to community edition. The instance of FusionAuth that makes this API request will be reconfigured, but each additional instance that should be licensed will need to be reconfigured with the updated license key.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Regenerate a Reactor license
+[.endpoint]
+.URI
+--
+[method]#PUT# [uri]#/api/reactor#
+--
+
+=== Response
+
+This API does not return a JSON response body.
+
+:never_search_error:
+include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
+:never_search_error!:
+
+
+== Deactivate Reactor license
+
+This API is used to deactivate a Reactor license. If link:/docs/v1/tech/reactor/#breached-password-detection[Breached Password Detection] was being used it will no longer work, licensed features can no longer be selected, and existing configurations may no longer work if they are configured using licensed features.
```

Comment:
```suggestion
 This API is used to deactivate a Reactor license. If link:/docs/v1/tech/reactor/#breached-password-detection[Breached Password Detection] was being used it will no longer work, licensed features can no longer be enabled, and existing configurations may no longer work if they use licensed features.
```